sp800-63c/sec6_presentation.md

```diff
@@ -38,11 +47,11 @@ In the *front-channel* model, the IdP creates an assertion and sends it to the s
 
 In the front-channel method, an assertion is visible to the subscriber, which could potentially cause leakage of system information included in the assertion. 
 
-Since the assertion is under the control of the subscriber, the direct method also allows the assertion to be replayed the subscriber to submit a single assertion to multiple RPs. This
+Since the assertion is under the control of the subscriber, the direct method also allows the subscriber to submit a single assertion to unintended parties, perhaps by a browser replaying an assertion at multiple RPs. Intentional use of this method can lead to lax audience restriction of the assertion itself, which in turn could lead to privacy and security breaches for the subscriber across these RPs. Such multi-RP use is not recommended. Instead, RPs are encouraged to fetch their own individual assertions.
```

Comment:
If it's restricted to multiple audiences (looseness on the server side) or the RP doesn't do the right checks (looseness on the client side) then this happens. It's not supposed to, of course, but in practice we see this a lot as RP deployers try to cut corners and get one assertion to use everywhere.